docs/write_test/development.md

```diff
@@ -10,13 +10,15 @@ what to expect of it).
 - [Coding guidelines](#coding-guidelines)
   - [Naming Conventions](#naming-conventions)
   - [Code checks](#code-checks)
-- [Document excellence](#document-excellence)
+  - [Test code excellence](#test-code-excellence)
   - [Code comments](#code-comments)
```

Comment:
The four parts are subtopics of document excellence. Is it changed?

Reply:
No, they are not changed. I only separated them into individual sections as I understood the "document excellence" for the document I am working on instead of logging/commits etc.